modules/pubCommonIdSystem.js

```diff
@@ -46,52 +140,78 @@ export const pubCommonIdSubmodule = {
    * decode the stored id value for passing to bid requests
    * @function
    * @param {string} value
+   * @param {SubmoduleConfig} config
    * @returns {{pubcid:string}}
    */
-  decode(value) {
-    return { 'pubcid': value }
+  decode(value, config) {
+    const sharedId = readData(config);
+    const idObj = {'pubcid': value};
+    if (sharedId) idObj['sharedid'] = {id: sharedId};
```

Comment:
very interesting...didn't realize this would "just work", cool.

Reply:
Yup, didn't realize it works this way, either.